lib/content-services/src/lib/tag/tags-creator/tags-creator.component.ts

```diff
@@ -0,0 +1,379 @@
+                finalize(() => (this._typing = false))
+            ).subscribe(({ exactResult, searchedResult }: {
+                exactResult: TagEntry;
+                searchedResult: TagPaging;
+            }) => {
+                if (exactResult) {
+                    this.existingExactTag = exactResult;
+                    this.removeExactTagFromSearchedResult(searchedResult);
+                    searchedResult.list.entries.unshift(exactResult);
+                } else {
+                    this.existingExactTag = null;
+                }
+
+                this._initialExistingTags = searchedResult.list.entries;
+                this.excludeAlreadyAddedTags(this._initialExistingTags);
+                this.exactTagSet$.next();
+                this._existingTagsLoading = false;
+            }, () => {
+                this.notificationService.showError('TAG.TAGS_CREATOR.ERRORS.FETCH_TAGS');
+                this._existingTagsLoading = false;
+            });
+        } else {
+            this.existingExactTag = null;
+        }
+    }
+
+    private removeExactTagFromSearchedResult(searchedResult: TagPaging): void {
+        const exactTagIndex = searchedResult.list.entries.findIndex(
+            (row) => this.compareTags(row.entry.tag, this.existingExactTag.entry.tag)
+        );
+
+        if (exactTagIndex > -1) {
+            searchedResult.list.entries.splice(exactTagIndex, 1);
+        }
+    }
+
+    private validateIfNotExistingTag(tagNameControl: FormControl<string>): Observable<TagNameControlErrors | null> {
+        return this.exactTagSet$.pipe(
+            map<void, TagNameControlErrors | null>(() =>
+                this.compareTags(tagNameControl.value, this.existingExactTag?.entry?.tag) ? { duplicatedExistingTag: true }
+                    : null
+            ), first()
+        );
+    }
+
+    private validateIfNotAlreadyAdded(tagNameControl: FormControl<string>): TagNameControlErrors | null {
+        return this.tags.some((tag) => this.compareTags(tag, tagNameControl.value))
+            ? { duplicatedAddedTag: true }
+            : null;
+    }
+
+    private compareTags(tagName1?: string, tagName2?: string): boolean {
+        return tagName1?.trim().toUpperCase() === tagName2?.trim().toUpperCase();
+    }
+
+    private validateEmptyTag(tagNameControl: FormControl<string>): TagNameControlErrors | null {
+        return tagNameControl.value.length && !tagNameControl.value.trim()
+            ? { emptyTag: true }
+            : null;
+    }
+
+    private setTagNameControlErrorMessageKey(): void {
+        this._tagNameErrorMessageKey = this.tagNameControl.invalid
+            ? `TAG.TAGS_CREATOR.ERRORS.${this.nameErrorMessagesByErrors.get(
+                  Object.keys(this.tagNameControl.errors)[0] as keyof TagNameControlErrors
+              )}`
+            : '';
+    }
+
+    private checkScrollbarVisibility(): void {
+        setTimeout(() => {
+            this._tagsListScrollbarVisible =
```

Comment:
Some time ago Denys asked me about the same when I raised original PR for ACC. Let me share link to my answer here :) 
https://github.com/Alfresco/alfresco-apps/pull/4759#discussion_r1088164224

So basically it is needed to align icons from list to all other icons (+ icon next to title and - icon next to input). In other case it gets not aligned when scrollbar becomes visible. Something like that: 
<img width="390" alt="image" src="https://user-images.githubusercontent.com/115619721/228160963-48e426b0-949a-4cb0-ad6c-cc42f3f838bc.png">

The reason is that there must be different space between icon and edge in that case because scrollbar takes extra space. 

Notice that this is not a case for ACA because there we won't have scrollbar. It is case for ACC mainly. The same is needed for categories I guess, in other case you will get not aligned icons as well